<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(51, 51, 51, 255), stop:1 rgba(25, 25, 25, 255));
color: #fff;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="controls">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="sortOptions">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>291</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: none;</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QRadioButton" name="hue">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Helvetica</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #fff;
padding: 5px;</string>
        </property>
        <property name="text">
         <string>Hue</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="saturation">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>102</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Helvetica</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #fff;
padding: 5px;</string>
        </property>
        <property name="text">
         <string>Saturation</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="contours">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>20</y>
          <width>102</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Helvetica</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #fff;
padding: 5px;
</string>
        </property>
        <property name="text">
         <string>Contours</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="sort">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>10</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
        </font>
       </property>
       <property name="text">
        <string>Sort</string>
       </property>
      </widget>
      <widget class="QPushButton" name="fileOpen">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
        </font>
       </property>
       <property name="text">
        <string>Open File</string>
       </property>
      </widget>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>0</y>
         <width>256</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="dirOpen">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
        </font>
       </property>
       <property name="text">
        <string>Open Directory</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QListWidget" name="results">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>653</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
